docs/src/app/components/Divider/README.md

```diff
@@ -0,0 +1,5 @@
+## Divider
+
+<a href="https://www.google.com/design/spec/components/dividers.html" target="_blank">Dividers</a> group and separate content within lists and page layouts. The divider is a thin rule, lightweight yet sufficient to distinguish content visually and spatially.
```

Comment:
What about `[Dividers](https://www.google.com/design/spec/components/dividers.html)` instead.
It would be better to not be linked to the dom to use https://github.com/lwansbrough/react-native-markdown later.